lib/modules/datasource/maven/index.ts

```diff
@@ -372,4 +241,44 @@ export class MavenDatasource extends Datasource {
 
     return result;
   }
+
+  @cache({
+    namespace: `datasource-maven`,
+    key: (
+      { registryUrl, packageName }: PostprocessReleaseConfig,
+      { version }: Release,
+    ) => `postprocessRelease:${registryUrl}:${packageName}:${version}`,
+    ttlMinutes: 24 * 60,
```

Comment:
In what scenarios do we expect this result to change?

Reply:
For example, we return `reject` for 500 error and cache this for one week, but actually five minutes later this http resource became `200 OK` again. We could distinguish these cases and use direct cache mechanism instead of decorator, but it complicates code. Also, I'd like to work towards an API for decorators which would allow to do these things.